.changeset/fluffy-parents-thank.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-css": patch
```

Comment:
This change will also affect the react package, so there should be a react changeset as well.

Reply:
Added a React changeset.